src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -368,9 +368,9 @@ def getFileSystem(
                 if self._fs_rotator.atCapacity():
                     self._fs_rotator.dropOldestFileSystem()
 
-                EnkfFs.createFileSystem(full_case_name)
-
-            new_fs = EnkfFs(full_case_name, read_only)
+                new_fs = EnkfFs.createFileSystem(full_case_name, read_only)
```

Comment:
Is this technically a bugfix? It might be that this breaks something due to relying on undocumented "wrong" behavior.

Reply:
I labeled it `breaking-change` because of that, but I grepped my whole environment and found not traces of its use in the usual suspects, so think it is safe to do.